src/Playwright/Core/FrameLocator.cs

```diff
@@ -39,9 +39,11 @@ public FrameLocator(Frame parent, string selector)
         _frameSelector = selector;
     }
 
-    IFrameLocator IFrameLocator.First => new FrameLocator(_frame, $"{_frameSelector} >> nth=0");
+    public IFrameLocator First => new FrameLocator(_frame, $"{_frameSelector} >> nth=0");
```

Comment:
Note: Explicit vs. Implicit interface implementation. No user-facing changes, just make it more consistent across the codebase.